spec/repositories/manual_repository_spec.rb

```diff
@@ -139,10 +139,19 @@
   end
 
   describe "#[]" do
+    let(:section_repository) { double(:section_repository) }
+
     before do
       allow(record_collection).to receive(:find_by).and_return(manual_record)
       allow(manual_record).to receive(:latest_edition).and_return(edition)
       allow(Manual).to receive(:new).and_return(manual)
+      allow(SectionRepository).to receive(:new).with(manual: manual).and_return(section_repository)
+      allow(manual).to receive(:'sections=')
+      allow(manual).to receive(:'removed_sections=')
+      allow(edition).to receive(:section_ids).and_return([:section_id])
+      allow(edition).to receive(:removed_section_ids).and_return([:removed_section_id])
+      allow(section_repository).to receive(:fetch).with(:section_id).and_return(:section)
+      allow(section_repository).to receive(:fetch).with(:removed_section_id).and_return(:removed_section)
```

Comment:
Yes - that's a good point. This style of testing is common to both the ManualRepository and SectionAssociationMarshaller specs before this PR, so I decided to move them across with as few changes as I could. But I think a good next step would be to make them less brittle (or to test their functionality in a different way if we move it to Manual)